_config.yml

```diff
@@ -4,8 +4,10 @@ permalink: /news/:year/:month/:day/:title/
 relative_permalinks: false
 markdown: kramdown
 highlighter: rouge
-site_env: prod
+site_env: demo
 env:
+    hj:  
```

Comment:
I'm not sure it's good to have dev machines in here.  Although it's unlikely anyone will use this open source project, it would be strange to have internal machines in here.

Reply:
removed.